luigi/worker.py

```diff
@@ -79,6 +79,10 @@
 _WAIT_INTERVAL_EPS = 0.00001
 
 
+def is_external(task):
```

Comment:
Maybe make it _private. Did you also consider putting it in task.py? No strong opinions though.